.github/workflows/release.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
           git push -f --tags
 
       - name: Release
-        uses: softprops/action-gh-release@v1
+        uses: hacbs-contract/action-gh-release@v2
```

Comment:
Can we add a TODO comment here? It would also be great to add a link to the upstream issue we're expecting to be resolved.

Reply:
Good point :)